### ISSUE TITLE
HTML/classic block inside column results in "modified externally" message masking all column

### ISSUE THREAD

**Issue by eddr** [WordPress/gutenberg#7401]
**Describe the bug**
1. using HTML/classic block inside column results in "This block appears to have been modified externally" message.
2. The message also masks all column surface, after the edit screen is reloaded (not immediately upon editing). That Doesn't happen by simply using these blocks outside of a column
3. Moreover, the message offers two ways to continue : transform to a block and edit as an HTML. Clicking any of these options results in disappearing of the inner block, even after reloading, while it is shown on the frontend..

**To Reproduce**
Create a column block. Create an HTML or classis editor blocks and place inside one of the columns.

**Expected behavior**
1. The error message shouldn't appear at all, there is no problem in editing the HTML of HTML block and since it's already a block, no need to transform it to block.
2. If there is any reason for it, then at least it shouldn't mask the whole column, as it blocks editing of other parts
3. If a block is removed from the edit screen, it shouldn't appear in the frontend

**Desktop (please complete the following information):**
 - OS: Windows 10 1803
 - Firefox 61b14

**Additional context**
- Gutenberg 3.0.1
- Using ACF 5.7RC2 plugin and that's it more or less

**Comment by sourourn** [comment]
I tried to reproduce the steps but was not able to see any error messages. 

Firefox (61.0.2) on MacOS
Gutenberg version: 4.5.1
WordPress version: 4.9.8

**Comment by dromero20** [comment]
I tried to replicate the issue but I didn't see any error message. 

- I created a column block.
- Added a classic editor block in one column and an HTML one on the other one.
- I edited both by adding a small <h1> text in each one. No error messages.

I also did the following:

- I created a column block.
- I created an HTML block and classic one and added <h1> line.
- Moved both blocks inside the column one.

No issues or warnings.

Tested in Firefox 63.0.1 (64-bit). macOS 10.14.
Gutenberg: 4.5.1
WordPress: 4.9.8
